roles/apache-perun/meta/main.yml

```diff
@@ -11,7 +11,7 @@ galaxy_info:
   - name: Debian
     versions:
     - stretch
-    - wheezy
+    - jessie
```

Comment:
I guess we want to support debian8+, but do we really need to be restrictive in install script. Is there currently any incompatibility ? I guess Apache config could be incompatible between wheezy and jessie.